app/page.tsx

```diff
@@ -1,7 +1,17 @@
+"use client";
+
+import SearchBar from "@/components/SearchBar";
+import React from "react";
+
 export default function Home() {
+  const [albumSearch, setAlbumSearch] = React.useState<string>("");
+
   return (
-    <main>
-      <h1>Home Page</h1>
+    <main className="flex flex-col items-center justify-center ">
+      <SearchBar
+        setAlbumSearch={(albumSearch) => setAlbumSearch(albumSearch)}
```

Comment:
it would be better DX for `SearchBar` to take a familiar prop (`onChange`) and would keep things more flexible too.

```suggestion
        onChange={(value) => setAlbumSearch(value)}
```